libs/ui-shared/src/lib/components/atoms/textArea/TextArea.js

```diff
@@ -0,0 +1,5 @@
+import { Input } from 'antd';
```

Comment:
textArea is defined in input above, please keep one

Reply:
I removed the separated one, as we will need a specific one which we added under input.